lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -334,6 +334,10 @@ def path_to_outcome
                    'opposite_sex'
                  end
 
+      if ceremony_country == 'philippines'
```

Comment:
Ireland only has two outcomes, but it's templates are at the top-level of the `outcomes` folder. We'll need to refactor the templates before we can add it to any `two outcomes` list.